semantic.cabal

```diff
@@ -304,6 +304,9 @@ library
                      , prettyprinter ^>= 1.2.1
                      , pretty-show ^>= 1.9.5
                      , profunctors ^>= 5.3
+                     , proto-lens == 0.5.1.0
+                     , proto-lens-jsonpb
```

Comment:
1. Do we need a version bound on this?
2. Should these version bounds be using `^>=` to express “up to the next major version”?

Reply:
Upon reflection we probably don’t need a version bound on `proto-lens-jsonpb` because we’re bringing it in with `source-repository-package`, but I think `^>=` for `proto-lens` is a good idea either way.

Reply:
We’ll eventually want to push `proto-lens-jsonpb` to Hackage so we can fix #16, so a version bound on both these is good to me.